src/Orders.sol

```diff
@@ -118,8 +128,7 @@ abstract contract OrderOrigin {
     /// @param token - The token to transfer.
     /// @custom:emits Sweep
     /// @custom:reverts OnlyBuilder if called by non-block builder
-    function sweep(address recipient, address token) public {
-        if (msg.sender != block.coinbase) revert OnlyBuilder();
+    function sweep(address recipient, address token) external {
```

Comment:
did we make a decision on specifying a min amount swept here?

Reply:
it isn't strictly necessary because `sweep` should be submitted in a bundle with the corresponding `initiate` calls; if another Filler gets their bundle submitted first, the second bundle's `initiate` calls will be rejected (because of the tx nonce) so the bundle should be rejected for that reason.